tbx/core/tests/test_navigation_field.py

```diff
@@ -0,0 +1,27 @@
+from tbx.core.factories import HomePageFactory, StandardPageFactory
+from wagtail.models import Site
+from wagtail.test.utils import WagtailPageTestCase
+
+
+class TestNavigationText(WagtailPageTestCase):
+    def setUp(self):
+        super().setUp()
+
+        site = Site.objects.get(is_default_site=True)
+        root = site.root_page.specific
+        self.home = HomePageFactory(parent=root)
+
+        site.root_page = self.home
+        site.save()
+
+        self.standard_page = StandardPageFactory(
+            title="Test standard page", parent=self.home
+        )
+
+    def test_override(self):
+        self.standard_page.navigation_text = "Test navigation override text"
+
+        self.assertEqual(self.standard_page.nav_text, "Test navigation override text")
```

Comment:
```suggestion
    def test_nav_text_prefers_navigation_tex(self):
        page = StandardPage(id=1, title="Title", navigation_text="Navigation text")
        self.assertEqual(page.nav_text, "Navigation text")
        self.assertEqual(page.nav_text, "Test navigation override text")
```

Reply:
NOTE: Specifying an `id` value prevents the following code from running in Wagtail, which errors when you don't have database access :P https://github.com/wagtail/wagtail/blob/stable/6.1.x/wagtail/models/__init__.py#L1336-L1339

Reply:
thanks for the explanation!